recipes-bsp/u-boot/files/pv.radxa.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_CMD_SETEXPR=y
```

Comment:
imo, we can setexpr to our default fragment. (pv.cfg) as we use setexpr in our default boot.cmd.pvgeneric

Reply:
Resolved.